java-engine/build.gradle.kts

```diff
@@ -2,10 +2,25 @@ plugins {
     `java-library`
     `maven-publish`
     signing
-    id("com.diffplug.spotless") version "6.23.2"
+    id ("com.diffplug.spotless").version("6.23.2")
     id("io.github.gradle-nexus.publish-plugin").version("2.0.0")
     id("pl.allegro.tech.build.axion-release").version("1.16.0")
     id("tech.yanand.maven-central-publish").version("1.3.0")
+    id("me.champeau.jmh").version("0.7.2")
+    id("at.released.wasm2class.plugin").version("0.3")
+}
+
+wasm2class {
+    modules {
+        create("Yggdrasil") {
+            wasm = file("../target/wasm32-unknown-unknown/release/pure_wasm.wasm")
+            targetPackage = "org.example.wasm"
```

Comment:
Seems like the wrong targetPackage? I'd think io.getunleash.wasm would be better?